test/sql/geo/ducklake_geometry_merge.test

```diff
@@ -0,0 +1,61 @@
+# name: test/sql/geo/ducklake_geometry_merge.test
+# group: [geo]
+
+require ducklake
+
+require parquet
+
+require spatial
+
+test-env DUCKLAKE_CONNECTION __TEST_DIR__/{UUID}.db
+
+statement ok
+ATTACH 'ducklake:${DUCKLAKE_CONNECTION}' AS ducklake (DATA_PATH '__TEST_DIR__/ducklake_files', METADATA_CATALOG 'ducklake_meta')
```

Comment:
Can you also add test-env DATA_PATH __TEST_DIR__ ?

As this will allow us to one day in the future run geo tests on a different storage.